DrivewayDentDeletion/Operators/cicd-dev/cicd-trigger-template.yaml

```diff
@@ -15,25 +15,25 @@ spec:
           - taskName: clone-git-source
             serviceAccountName: pipeline
           - taskName: build-mq
-            serviceAccountName: cicd-mq-sa
+            serviceAccountName: pipeline
           - taskName: deploy-wait-mq
-            serviceAccountName: cicd-mq-sa
+            serviceAccountName: pipeline
           - taskName: build-ace-int-server-ace-api
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: build-ace-int-server-ace-bernie
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: build-ace-int-server-ace-acme
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: build-ace-int-server-ace-chris
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: deploy-wait-ace-api
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: deploy-wait-ace-chris
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: deploy-wait-ace-acme
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
           - taskName: deploy-wait-ace-bernie
-            serviceAccountName: cicd-ace-sa
+            serviceAccountName: pipeline
```

Comment:
Since everything is now using the same service account could change from a map and specify a single service account by changing `serviceAccountNames` to `serviceAccountName`

Reply:
Note this applies to all 3 pipelines

Reply:
Done